CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 <!-- ## [Unreleased] -->
 
+## [5.1.1] - 2018-07-09
+- Rename `plugin/content-no-strings` rule to `shopify/content-no-strings`.
```

Comment:
This is a breaking change :/ Unless you mark 5.1.0 as yanked, like so:

```
## 5.1.0 - 2018-07-05 [YANKED]

Use 5.1.1 instead.
```

And pull all changelog items from 5.1.0 to 5.1.1.

Reply:
Done. 😬 Worth it in the end, I think